content/docs/fundamentals/converting/resolving-errors@ko.md

```diff
@@ -0,0 +1,350 @@
+
+```html
+<link rel="canonical" href="/article.html">
+```
+
+{% call callout('참고', type='note') %}
+독립된 표준 AMP 페이지도 만들 수 있습니다. 여전히 표준 링크는 필요하며 AMP 문서 자체를 가리켜야 합니다.
+
+```html
+<link rel="canonical" href="article.amp.html">
+```
+{% endcall %}
+
+이제 페이지를 **새로고침**합니다. 아직 수정해야 할 오류가 많지만 표준 링크 오류는 모두 사라졌습니다.
+
+## AMP 속성 지정
+
+페이지를 AMP 문서로 선언하려면 페이지의 루트 `<html>` 요소에 속성이 필요합니다.
+
+<pre class="error-text">
+'html ⚡ for top-level html' 태그에서 필수 '⚡' 속성이 누락되었습니다.
+필수 'html ⚡ for top-level html' 태그가 누락되었거나 잘못되었습니다.
+</pre>
+
+위에 표시된 오류는 다음과 같이 `<html>` 태그에 `⚡` 속성을 추가하면 간단히 해결됩니다.
+
+```html
+<html ⚡ lang="en">
+```
+
+이제 페이지를 새로고침하고 두 오류가 모두 사라졌는지 확인합니다.
+
+{% call callout('참고', type='note') %}
+`⚡`을 지정하는 것이 권장되는 방법이지만 다음과 같이 `⚡` 속성 자리에 `amp` 속성을 사용할 수도 있습니다.
+
+```html
+<html amp lang="en">
+```
+{% endcall %}
+
+## 표시 영역 지정
+
+이제 다음 오류를 해결해 보겠습니다.
+
+<pre class="error-text">
+필수 'meta name=viewport' 태그가 누락되었거나 잘못되었습니다.
+</pre>
+
+AMP에서는 표시 영역의 `width` 및 `minimum-scale`을 정의해야 합니다. 이 값은 각각 `device-width` 및 `1`로 정의되어야 합니다. 표시 영역은 HTML 페이지의 `<head>`에 포함되어 있는 일반적인 태그입니다.
+
+표시 영역 오류를 해결하려면 다음의 HTML 스니펫을 `<head>` 태그에 추가합니다.
+
+```html
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+```
+
+`width` 및 `minimum-scale`에 지정된 값은 AMP에서 요구되는 값입니다. `initial-scale` 정의는 필수 항목은 아니지만 모바일 웹 개발 과정에 일반적으로 포함되며 권장되는 사항입니다. [표시 영역 구성](https://developers.google.com/speed/docs/insights/ConfigureViewport)에서 표시 영역 및 반응형 디자인에 관해 자세히 알아볼 수 있습니다.
+
+이전과 마찬가지로 페이지를 **새로고침**하고 오류가 사라졌는지 확인합니다.
+
+## 외부 스타일시트 교체
+
+다음의 오류는 스타일시트 사용과 관련되어 있습니다.
+
+<pre class="error-text">
+'link rel=stylesheet for fonts' 태그에 있는 'href' 속성이 잘못된 값인 'base.css'로 설정되어 있습니다.
+</pre>
+
+특히 이 오류는 `<head>` 태그에 있는 다음과 같은 스타일시트 링크 태그와 관련되어 있습니다.
+
+```html
+<link href="base.css" rel="stylesheet" />
+```
+
+문제는 외부 스타일시트를 참조한 데서 발생합니다. 문서를 가능한 한 빨리 로드하기 위해 AMP에는 외부 스타일시트를 포함할 수 없습니다. 대신 모든 스타일시트 규칙은 `<style amp-custom></style>` 태그를 사용하여 AMP 문서에 인라인으로 추가되어야 합니다.
+
+```html
+<style amp-custom>
+
+/* The content from base.css */
+
+</style>
+```
+
+이제 오류를 해결해 보겠습니다.
+
+1.  `<head>`에서 외부 스타일시트를 가리키는 `<link>` 태그를 **삭제**하고 이를 인라인 `<style amp-custom></style>` 태그로 교체합니다. 스타일 태그의 `amp-custom` 속성은 필수입니다.
+2. [`base.css`](https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.css) 파일에 있는 모든 스타일을 **복사**하여 `<style amp-custom></style>` 태그에 붙여넣습니다.
+
+다시 한번 페이지를 **새로고침**하고 스타일시트 오류가 사라졌는지 확인합니다.
+
+{% call callout('참고', type='note') %}
+인라인 스타일링이 필요할 뿐 아니라 모든 스타일링 정보의 파일 크기가 50KB로 제한됩니다. AMP 페이지에 CSS를 인라인 처리하기 전에 [SASS](http://sass-lang.com/)와 같은 CSS 사전 처리기를 사용하여 CSS를 축소해야 합니다.
+{% endcall %}
+
+{% call callout('중요', type='caution') %}
+전체 AMP 문서에 대해 스타일 태그를 1개만 사용할 수 있습니다. AMP 페이지에서 참조하는 외부 스타일시트가 여러 개인 경우 이러한 스타일시트를 하나의 규칙 세트로 조합해야 합니다. AMP에서 어떤 CSS 규칙이 유효한지 알아보려면 [지원되는 CSS](/ko/docs/design/responsive/style_pages.html)를 확인하세요.
+{% endcall %}
+
+## 타사 자바스크립트 제외
+
+CSS를 인라인 처리하면 스타일시트를 AMP로 비교적 쉽게 다시 작성할 수 있지만 자바스크립트에서는 그렇지 않습니다.
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).